app/helpers/users_helper.rb

```diff
@@ -16,9 +16,10 @@ def comment_commentable_title(comment)
     if commentable.nil?
       deleted_commentable_text(comment)
     elsif commentable.hidden?
-      "<abbr title='#{deleted_commentable_text(comment)}'>".html_safe +
-      commentable.title +
-      "</abbr>".html_safe
+      "<del>".html_safe + commentable.title.html_safe + "</del>".html_safe + ' ' +
```

Comment:
What do you think about using [content_tag](http://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag)? Like:
```
content_tag(:del, commentable.title) + ' ' +
  content_tag(:span, deleted_commentable_text(comment), class: 'small')
```